Toolset/palettes/message box/behaviors/revmessageboxbehavior.livecodescript

```diff
@@ -587,7 +587,7 @@ command ideMessageBoxExecuteScript pScript
       put empty into tIntelligenceObject
    end try
    local tDebugMode
-   put revIDEGetPreference("IDEDebugMode") into tDebugMode
+   put the traceStack is not empty into tDebugMode
```

Comment:
@livecodeali this could be true when profiling... not sure what to suggest here other than again determining if there's an open script editor in debug mode.